db/schema.rb

```diff
@@ -377,6 +377,7 @@
   create_table "efile_submissions", force: :cascade do |t|
     t.datetime "created_at", precision: 6, null: false
     t.string "irs_submission_id"
+    t.datetime "last_checked_for_ack_at"
```

Comment:
i'd love to see this somewhere on the activity log list... maybe we could make a "synthetic log" to inject this onto the history list?

not as part of this work obviously, but sometime soon.